<ui version="4.0" >
 <class>RoseAppleClass</class>
 <widget class="QMainWindow" name="RoseAppleClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>RoseApple</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="RoseApple.qrc" >
    <normaloff>:/RoseApple/Resources/icon.png</normaloff>:/RoseApple/Resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QMdiArea" name="mdiArea" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase" >
    <property name="title" >
     <string>Database</string>
    </property>
    <addaction name="actionConnectDB" />
    <addaction name="separator" />
    <addaction name="actionDBTableView" />
    <addaction name="actionQuery" />
    <addaction name="actionDataImpot" />
    <addaction name="actionSQL_Maker" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuWindows" >
    <property name="title" >
     <string>Windows</string>
    </property>
    <addaction name="actionTappedView" />
    <addaction name="actionSubwindow_View" />
    <addaction name="separator" />
    <addaction name="actionCascadeView" />
    <addaction name="actionTileView" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionCSTextManager" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuDatabase" />
   <addaction name="menuTools" />
   <addaction name="menuWindows" />
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnectDB" />
   <addaction name="separator" />
   <addaction name="actionDBTableView" />
   <addaction name="actionQuery" />
   <addaction name="actionDataImpot" />
   <addaction name="actionSQL_Maker" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionTappedView" />
   <addaction name="actionSubwindow_View" />
   <addaction name="separator" />
   <addaction name="actionCascadeView" />
   <addaction name="actionTileView" />
  </widget>
  <widget class="QToolBar" name="toolBar_2" >
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionCSTextManager" />
  </widget>
  <action name="actionSQL_Maker" >
   <property name="icon" >
    <iconset resource="RoseApple.qrc" >
     <normaloff>:/RoseApple/Resources/notes.png</normaloff>:/RoseApple/Resources/notes.png</iconset>
   </property>
   <property name="text" >
    <string>SQL Maker</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="RoseApple.qrc" >
     <normaloff>:/RoseApple/Resources/exit.png</normaloff>:/RoseApple/Resources/exit.png</iconset>
   </property>
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCascadeView" >
   <property name="icon" >
    <iconset resource="RoseApple.qrc" >
     <normaloff>:/RoseApple/Resources/cascade.png</normaloff>:/RoseApple/Resources/cascade.png</iconset>
   </property>
   <property name="text" >
    <string>CascadeView</string>
   </property>
   <property name="iconText" >
    <string>Cascade View</string>
   </property>
  </action>
  <action name="actionTileView" >
   <property name="icon" >
    <iconset resource="RoseApple.qrc" >
     <normaloff>:/RoseApple/Resources/tile.png</normaloff>:/RoseApple/Resources/tile.png</iconset>
   </property>
   <property name="text" >
    <string>Tile View</string>
   </property>
  </action>
  <action name="actionTappedView" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="RoseApple.qrc" >
     <normaloff>:/RoseApple/Resources/tab.png</normaloff>:/RoseApple/Resources/tab.png</iconset>
   </property>
   <property name="text" >
    <string>TappedView</string>
   </property>
   <property name="iconText" >
    <string>Tapped View</string>
   </property>
  </action>
  <action name="actionClose_All" >
   <property name="icon" >
    <iconset resource="RoseApple.qrc" >
     <normaloff>:/RoseApple/Resources/delete.png</normaloff>:/RoseApple/Resources/delete.png</iconset>
   </property>
   <property name="text" >
    <string>Close All</string>
   </property>
  </action>
  <action name="actionSubwindow_View" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="RoseApple.qrc" >
     <normaloff>:/RoseApple/Resources/subwin.png</normaloff>:/RoseApple/Resources/subwin.png</iconset>
   </property>
   <property name="text" >
    <string>Subwindow View</string>
   </property>
  </action>
  <action name="actionConnectDB" >
   <property name="icon" >
    <iconset resource="RoseApple.qrc" >
     <normaloff>:/RoseApple/Resources/new.png</normaloff>:/RoseApple/Resources/new.png</iconset>
   </property>
   <property name="text" >
    <string>Connect Database</string>
   </property>
  </action>
  <action name="actionDBTableView" >
   <property name="icon" >
    <iconset resource="RoseApple.qrc" >
     <normaloff>:/RoseApple/Resources/set.png</normaloff>:/RoseApple/Resources/set.png</iconset>
   </property>
   <property name="text" >
    <string>DB Table View</string>
   </property>
  </action>
  <action name="actionDataImpot" >
   <property name="icon" >
    <iconset resource="RoseApple.qrc" >
     <normaloff>:/RoseApple/Resources/import.png</normaloff>:/RoseApple/Resources/import.png</iconset>
   </property>
   <property name="text" >
    <string>Import Data</string>
   </property>
  </action>
  <action name="actionCSTextManager" >
   <property name="icon" >
    <iconset resource="RoseApple.qrc" >
     <normaloff>:/RoseApple/Resources/atext24.png</normaloff>:/RoseApple/Resources/atext24.png</iconset>
   </property>
   <property name="text" >
    <string>CS Text Manager</string>
   </property>
  </action>
  <action name="actionQuery" >
   <property name="icon" >
    <iconset resource="RoseApple.qrc" >
     <normaloff>:/RoseApple/Resources/query.png</normaloff>:/RoseApple/Resources/query.png</iconset>
   </property>
   <property name="text" >
    <string>Query</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="RoseApple.qrc" />
 </resources>
 <connections/>
</ui>
